package.json

```diff
@@ -2,25 +2,18 @@
 	"name": "package-json-validator",
 	"version": "0.10.0",
 	"description": "Tools to validate package.json files.",
-	"keywords": [
-		"package.json",
-		"validator",
-		"package.json validator",
-		"lint",
-		"package.json linter"
-	],
```

Comment:
🐛 These should not have been deleted. https://github.com/JoshuaKGoldberg/create-typescript-app/issues/2094